packages/client/src/components/add-liquidity/add-liquidity-v3.tsx

```diff
@@ -0,0 +1,129 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLink } from '@fortawesome/free-solid-svg-icons';
+import { TokenInput } from 'components/token-input';
+import { WalletBalance } from 'components/wallet-balance';
+import { IUniswapPair } from '@sommelier/shared-types';
+import { Range } from 'rc-slider';
+import { WalletBalances } from 'types/states';
+import 'rc-slider/assets/index.css';
+import {
+    Form,
+    Row,
+    FormControl,
+    InputGroup,
+    Dropdown,
+    DropdownButton,
+} from 'react-bootstrap';
+import { faRetweet } from '@fortawesome/free-solid-svg-icons';
+type Props = {
+    balances: WalletBalances;
+    pairId: string | null;
+    pairData: IUniswapPair | null;
+};
+export const AddLiquidityV3 = ({
+    balances,
+    pairId,
+    pairData,
+}: Props): JSX.Element => {
+    
+    const [token0Amount, setToken0Amount] = useState('0');
+    const [token1Amount, setToken1Amount] = useState('0');
+
+    const token0Logo = resolveLogo(pairData?.token0.id);
+    const token1Logo = resolveLogo(pairData?.token1.id);
+    const token0 = pairData?.token0.symbol ?? '';
+    const token1 = pairData?.token1.symbol ?? '';
+
+    return (
+        <>
+            <div className='add-v3-container'>
+                <div className='token-and-wallet'>
+                    <div className='token-pair-selector'>
+                        <TokenInput
+                            token={token0}
+                            amount={token0Amount}
+                            updateAmount={setToken0Amount}
+                            updateToken={() => {
+                                return '';
+                            }}
+                            handleTokenRatio={() => {
+                                return '';
+                            }}
+                            options={['ETH', token0]}
+                            balances={balances}
+                            twoSide={false}
+                        />
+                        <FontAwesomeIcon icon={faRetweet} />
+
+                        <TokenInput
+                            token={token1}
+                            amount={token1Amount}
+                            updateAmount={setToken1Amount}
+                            updateToken={() => {
+                                return '';
+                            }}
+                            handleTokenRatio={() => {
+                                return '';
+                            }}
+                            options={['ETH', token1]}
+                            balances={balances}
+                            twoSide={true}
+                        />
+                    </div>
+                    <div className='wallet-fees'>
+                        <WalletBalance balances={balances} />
+                    </div>
+                </div>
+                <br />
+                {/* <div className='tab-container'>
```

Comment:
Are these the alternative strategies commented out so we end up with only one option?

Reply:
yes, i commented it out and left it in there for now. When we are ready with the feature set i will do a clean up and remove all the deadcode.